ENGyn.NodesTestPlatform/ENGyn.NodesTestPlatform/Providers/RunnableProvider.cs

```diff
@@ -0,0 +1,96 @@
+﻿using ENGyn.NodesTestPlatform.Core;
+using ENGyn.NodesTestPlatform.Models;
+using ENGyn.NodesTestPlatform.Services;
+using ENGyn.NodesTestPlatform.Utils;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ENGyn.NodesTestPlatform.Providers
+{
+    public class RunnableProvider : IRunnableService
+    {
+        private readonly string _art;
+        private readonly ReflectionHandler _reflectionHandler;
```

Comment:
Please add an interface `IReflectionHandler` so that `ReflectionHandler` can implement it. Do the same for `CommandValidation` and then replace it here. In general (except for models) all classes should implement at least one interface that contains the corresponding methods and signatures. This will allow us to create unit tests later without the need of refactor.

Reply:
Implemented. Both classes now implement their respective interfaces containing their corresponding methods and signatures. Also i decided to move these classes under `providers` namespace because is where the implementations of these services live.